packages/esm-patient-banner-app/src/banner-tags/deceased-tag.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import DeceasedBannerTag from './deceased-tag.component';
+import { mockDeceasedPatient } from '../../../../__mocks__/mockDeceasedPatient';
+import { mockPatient } from '../../../../__mocks__/patient.mock';
+
+describe('deceasedTag', () => {
+  it('renders a deceased tag in the patient banner for patients who died', () => {
+    renderDeceasedBannerTag();
+    expect(screen.getByRole('button', { name: /Deceased/ })).toBeInTheDocument();
+  });
+
+  it('doesnot render Deceased tag for patients who are still alive', () => {
+    alivePatient();
+    expect(screen.queryByRole('button', { name: /Deceased/ })).not.toBeInTheDocument();
+  });
+});
+
+function renderDeceasedBannerTag() {
+  render(<DeceasedBannerTag patient={mockDeceasedPatient} />);
+}
+
+function alivePatient() {
+  render(<DeceasedBannerTag patient={mockPatient} />);
+}
```

Comment:
There's no point in making one line functions that are only used once. They just make the code more complex. Please inline them.

Reply:
That is exactly what i felt when i was looking at this!

Reply:
noted